kubejs/startup_scripts/worldgen_remove.js

```diff
@@ -1,8 +1,14 @@
 WorldgenEvents.remove((event) => {
     // https://mods.latvian.dev/books/kubejs/page/worldgen-events
     // console.debugEnabled = true;
-    // event.removeOres((props) => {
-    //     props.worldgenLayer = 'underground_ores';
-    //     props.blocks = ['blue_skies:horizonite_ore', 'blue_skies:ventium_ore', 'blue_skies:falsite_ore'];
-    // });
+    event.removeOres((props) => {
+        props.worldgenLayer = 'underground_ores';
+        props.blocks = [
+            'blue_skies:horizonite_ore',
+            'blue_skies:ventium_ore',
+            'blue_skies:falsite_ore',
+            'occultism:silver_ore',
+            'occultism:silver_ore_deepslate'
```

Comment:
Ametrine and Pendorite (Biomes you'll go) can also be yeeted and all related items hidden in JEI